ReactAndroid/src/main/java/com/facebook/react/bridge/Arguments.java

```diff
@@ -277,7 +315,57 @@ public static WritableMap fromBundle(Bundle bundle) {
   }
 
   /**
+   * Convert a {@link WritableArray} to a {@link ArrayList}.
+   *
+   * @param readableArray the {@link WritableArray} to convert.
+   * @return the converted {@link ArrayList}.
+   */
+  @Nullable
+  public static ArrayList toList(@Nullable ReadableArray readableArray) {
```

Comment:
Can you make the return type List? It's better to return the abstract interface rather than the concrete implementation type.